ocaml/testsuite/tests/typing-layouts/mixed_constructor_arguments.ml

```diff
@@ -15,11 +15,7 @@ type t_ext = ..
 type t_cstr_boxed_float = A of float * float#
```

Comment:
I can't leave a comment up at the top, but I think we should set this test up to also run under `beta` and `alpha`, now that we're deleting the other file.  See the test stanzas in the other changed files in this PR for how.